Apply edits based on bemasc's review (#380)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-protocol.md

Issuer directory resources have the media type
"application/token-issuer-directory" and are located at the well-known location
/.well-known/token-issuer-directory; see {{wkuri-reg}} for the registration
information for this well-known URI. The reason that this resource is located

> at a well-known URI is that Issuers are defined by their origin in TokenChallenge

Reviewer: ```suggestion
at a well-known URI is that Issuers are defined by an origin name in TokenChallenge
```

result in use of stale Issuer configuration information, whereas short
lifetimes may result in decreased performance. When use of an Issuer
configuration results in token issuance failures, e.g., because the

> Issuer directory resource is no longer valid and issuance requests using this

Reviewer 2: It sounds like you're saying that the directory resource is fresh but invalid?  If so, it would be clearer to make that explicit.
```suggestion
Issuer has invalidated its directory resource before its expiration time and issuance requests using this
```

Author: Yes, exactly, thanks for the suggestion!
